bindings/java/src/main/native/Makefile.am

```diff
@@ -9,7 +9,7 @@ jardir         = $(libdir)
 topdir         = $(abs_top_builddir)
 java_build_dir = $(builddir)/build-java
 maven_repo    ?= $(java_build_dir)/.deps
-jarfile        = $(java_build_dir)/jucx-@VERSION@.jar
+jarfile        = $(java_build_dir)/jucx-@VERSION@-$(arch).jar
```

Comment:
do we want such suffix also for x86_64?

Reply:
Don't we?

Reply:
After consulting with stakeholders, decided to retain the original filename.
Pushed another commit.